_build/templates/default/sass/_colors-and-vars.scss

```diff
@@ -88,19 +89,14 @@ $buttonColorPrimary: $white;
 $buttonColorPrimaryHover: $white;
 $buttonColorPrimaryActive: $white;
 $buttonColorPrimaryDisabled: $white;
-$buttonPrimaryBg1: $primary1;
-$buttonPrimaryBg2: $primary2;
-$buttonPrimaryBg1Hover: darken($buttonPrimaryBg1, 6%);
-$buttonPrimaryBg2Hover: darken($buttonPrimaryBg2, 6%);
-$buttonPrimaryBg1Active: darken($buttonPrimaryBg1, 12%);
-$buttonPrimaryBg2Active: darken($buttonPrimaryBg2, 12%);
+$buttonPrimaryBgActive: darken($green, 12%);
```

Comment:
Make sense to use `$darkGreen` variable here.

Reply:
Thanks